crates/solidity/testing/snapshots/cst_output/Block/unchecked/generated/0.4.11-failure.yml

```diff
@@ -5,20 +5,22 @@ Source: >
 
 Errors: # 1 total
   - >
-    Error: Expected Ampersand or AmpersandAmpersand or AmpersandEqual or Asterisk or AsteriskAsterisk or AsteriskEqual or BangEqual or Bar or BarBar or BarEqual or Caret or CaretEqual or Equal or EqualEqual or GreaterThan or GreaterThanEqual or GreaterThanGreaterThan or GreaterThanGreaterThanEqual or GreaterThanGreaterThanGreaterThan or GreaterThanGreaterThanGreaterThanEqual or LessThan or LessThanEqual or LessThanLessThan or LessThanLessThanEqual or Minus or MinusEqual or Percent or PercentEqual or Plus or PlusEqual or Semicolon or Slash or SlashEqual.
-       ╭─[crates/solidity/testing/snapshots/cst_output/Block/unchecked/input.sol:1:12]
+    Error: Unrecognized sequence.
+       ╭─[crates/solidity/testing/snapshots/cst_output/Block/unchecked/input.sol:1:13]
        │
      1 │ { unchecked { x = 1; } }
-       │            ──────┬──────  
-       │                  ╰──────── Error occurred here.
+       │             ───┬───  
+       │                ╰───── Error occurred here.
     ───╯
 
 Tree:
-  - Block (Rule): # 0..24 "{ unchecked { x = 1; } }"
+  - Block (Rule): # 0..22 "{ unchecked { x = 1; }"
```

Comment:
The `Block` changed here from 24 to 22 bytes, as we dropped the last `}`. Is that intentional?

The CST should always be complete (covers the entire input).

Reply:
I can bring back the SKIPPED token handling for the incomplete parse